src/core/imap/MCIMAPSession.cpp

```diff
@@ -2813,8 +2813,9 @@ static void nstringDeallocator(char * bytes, unsigned int length) {
     mailimap_nstring_free(bytes);
 };
 
-Data * IMAPSession::fetchMessageAttachment(String * folder, bool identifier_is_uid,
+Data * IMAPSession::fetchNonDecodedMessageAttachment(String * folder, bool identifier_is_uid,
```

Comment:
recreate fetchMessageAttachment() that will use fetchNonDecodedMessageAttachment().